docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Documentation
```

Comment:
could you put a link to this in the main (root) readme as well please

Reply:
Done. See comment below👇